package.json

```diff
@@ -14,24 +14,31 @@
   "author": "",
   "license": "BSD-3-Clause",
   "devDependencies": {
-    "@11ty/eleventy": "^0.3.5",
-    "autoprefixer": "^8.6.3",
+    "@11ty/eleventy": "^0.7.1",
+    "@babel/core": "^7.4.0",
+    "@babel/plugin-external-helpers": "^7.2.0",
+    "autoprefixer": "^9.5.0",
     "babel-core": "^6.26.3",
-    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-external-helpers": "^7.0.0-beta.3",
     "babel-preset-env": "^1.7.0",
-    "browser-sync": "^2.24.4",
-    "gulp": "^3.9.1",
+    "browser-sync": "^2.26.3",
+    "gulp": "^4.0.0",
+    "gulp-babel": "^8.0.0",
     "gulp-cssnano": "^2.1.3",
-    "gulp-header": "^2.0.5",
-    "gulp-postcss": "^7.0.1",
-    "gulp-rename": "^1.3.0",
-    "gulp-sass": "^4.0.1",
-    "gulp-uglify": "^3.0.0",
+    "gulp-header": "^2.0.7",
+    "gulp-postcss": "^8.0.0",
+    "gulp-rename": "^1.4.0",
+    "gulp-sass": "^4.0.2",
+    "gulp-uglify": "^3.0.2",
     "gulp-util": "^3.0.8",
     "pump": "^3.0.0",
-    "rivet-uits": "^1.1.0",
-    "rollup": "^0.60.4",
-    "rollup-plugin-babel": "^3.0.4",
-    "run-sequence": "^2.2.1"
+    "rivet-uits": "^1.3.0",
+    "rollup": "^1.7.3",
+    "rollup-plugin-babel": "^4.3.2",
+    "run-sequence": "^2.2.1",
+    "tslib": "^1.9.3"
+  },
+  "dependencies": {
+    "rollup-plugin-typescript": "^1.0.1"
```

Comment:
We aren't using typescript in any of our JS, so we shouldn't need this as a dependency or devDependency. I think once you address the above issues it should take care of this.